.github/workflows/_wiki-documentation.yaml

```diff
@@ -0,0 +1,85 @@
+# This reusable workflow publishes Markdown docs to Github Wiki. Some manual
+# setup is required before using it: enable Wiki in repository and create at
+# least one page.
+
+name: Publish Github Wiki documentation
+on:
+  workflow_call:
+    inputs:
+      release_tag:
+        required: true
+        type: string
+      release_url:
+        required: true
+        type: string
+      artifact_name:
+        required: true
+        type: string
+      git_user_name:
+        required: true
+        type: string
+      git_user_email:
+        required: true
+        type: string
+    secrets:
+      REPO_ACCESS_TOKEN:
+        required: true
+
+permissions:
+  contents: read
+
+jobs:
+  publish-wiki:
+    name: Publish Github Wiki
+    if: github.repository.has_wiki == true
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Harden Runner
+      uses: step-security/harden-runner@2e205a28d0e1da00c5f53b161f4067b052c61f34 # v1.5.0
+      with:
+        egress-policy: audit # TODO: change to 'egress-policy: block' after couple of runs
+
+    # Check out the repository's Wiki repo into the wiki/ folder. The token is required
+    # only for private repositories.
```

Comment:
If the token is only required for private repositories, why not to comment it here and ask users to uncomment if they have a private repo

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/396/commits/0ee1aa67dd8572618b21836f8d5f0017b35740f4.

Reply:
That’s commenting/un-commenting in three places then…

https://github.com/jenstroeger/python-package-template/blob/798786b40760032dc0f218e8ac630a7f51739d7a/.github/workflows/release.yaml#L246-L247

https://github.com/jenstroeger/python-package-template/blob/798786b40760032dc0f218e8ac630a7f51739d7a/.github/workflows/_wiki-documentation.yaml#L24-L26

https://github.com/jenstroeger/python-package-template/blob/798786b40760032dc0f218e8ac630a7f51739d7a/.github/workflows/_wiki-documentation.yaml#L47-L48

Not sure it’s worthwhile, and maybe it’s just easier to always ask for the token?

Reply:
And one more note: the `REPO_ACCESS_TOKEN` is also required by the Release workflow to push the bump commit, so we’re not asking anything new.